src/commands/crashes/category.txt

```diff
@@ -0,0 +1 @@
+Commands related to Crashes
```

Comment:
@anlinde, @olegd Could you, please, provide the help message for the "crashes" command category?